module/dkg/utils.go

```diff
@@ -0,0 +1,22 @@
+package dkg
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func GetDKGCommitteeIndex(nodeID flow.Identifier, participants flow.IdentityList) (int, error) {
```

Comment:
Can we move this function to `model/flow/identity.go` to be something like `GetIndex(flow.Identifier) (uint, bool)`?

Reply:
Thanks for this

Reply:
https://github.com/onflow/flow-go/pull/1849/commits/aa369fc7718a37db5db1e76cb77481bc0c1cca33